Test: randoop.xmlsecurity.RandoopTest1.test124
Intended behavior: FAILURE
when executed after: [randoop.xmlsecurity.RandoopTest1.test13]
The revealed different behavior: PASS
when executed after: []

Test: randoop.xmlsecurity.RandoopTest0.test213
Intended behavior: FAILURE
when executed after: [randoop.xmlsecurity.RandoopTest0.test138]
The revealed different behavior: PASS
when executed after: []

Test: randoop.xmlsecurity.RandoopTest1.test13
Intended behavior: FAILURE
when executed after: [randoop.xmlsecurity.RandoopTest0.test213]
The revealed different behavior: PASS
when executed after: []

Test: randoop.xmlsecurity.RandoopTest0.test138
Intended behavior: FAILURE
when executed after: [randoop.xmlsecurity.RandoopTest0.test41]
The revealed different behavior: PASS
when executed after: []

Test: randoop.xmlsecurity.RandoopTest0.test41
Intended behavior: FAILURE
when executed after: [randoop.xmlsecurity.RandoopTest0.test40]
The revealed different behavior: PASS
when executed after: []

Test: randoop.xmlsecurity.RandoopTest0.test28
Intended behavior: FAILURE
when executed after: [randoop.xmlsecurity.RandoopTest0.test15]
The revealed different behavior: PASS
when executed after: []

